easybuild/easyconfigs/g/gmvapich2/gmvapich2-2016a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = "Toolchain"
+
+name = 'gmvapich2'
+version = '2016a'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain,
+ including MVAPICH2 for MPI support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compname = 'GCC'
+compver = '4.9.3-2.25'
+comp = (compname, compver)
+
+mpilib = 'MVAPICH2'
+mpiver = '2.2b'
```

Comment:
these are used in only one place, so just drop `mpilib` and `mpiver`, and include directly in `dependencies`?